tests/compile_ouroboros.py

```diff
@@ -0,0 +1,670 @@
+    'ouroboros/subprocess.py',
+    'ouroboros/sunau.py',
+    'ouroboros/symbol.py',
+    'ouroboros/symtable.py',
+    'ouroboros/sysconfig.py',
+    'ouroboros/telnetlib.py',
+    'ouroboros/tempfile.py',
+    'ouroboros/textwrap.py',
+    'ouroboros/this.py',
+    'ouroboros/threading.py',
+    'ouroboros/tkinter/__main__.py',
+    'ouroboros/tkinter/_fix.py',
+    'ouroboros/tkinter/colorchooser.py',
+    'ouroboros/tkinter/commondialog.py',
+    'ouroboros/tkinter/constants.py',
+    'ouroboros/tkinter/dialog.py',
+    'ouroboros/tkinter/dnd.py',
+    'ouroboros/tkinter/filedialog.py',
+    'ouroboros/tkinter/messagebox.py',
+    'ouroboros/tkinter/simpledialog.py',
+    'ouroboros/tkinter/tix.py',
+    'ouroboros/token.py',
+    'ouroboros/tokenize.py',
+    'ouroboros/tracemalloc.py',
+    'ouroboros/tty.py',
+    'ouroboros/turtledemo/bytedesign.py',
+    'ouroboros/turtledemo/chaos.py',
+    'ouroboros/turtledemo/clock.py',
+    'ouroboros/turtledemo/colormixer.py',
+    'ouroboros/turtledemo/forest.py',
+    'ouroboros/turtledemo/fractalcurves.py',
+    'ouroboros/turtledemo/lindenmayer.py',
+    'ouroboros/turtledemo/minimal_hanoi.py',
+    'ouroboros/turtledemo/nim.py',
+    'ouroboros/turtledemo/paint.py',
+    'ouroboros/turtledemo/peace.py',
+    'ouroboros/turtledemo/penrose.py',
+    'ouroboros/turtledemo/planet_and_moon.py',
+    'ouroboros/turtledemo/round_dance.py',
+    'ouroboros/turtledemo/tree.py',
+    'ouroboros/turtledemo/two_canvases.py',
+    'ouroboros/turtledemo/wikipedia.py',
+    'ouroboros/turtledemo/yinyang.py',
+    'ouroboros/types.py',
+    'ouroboros/unittest/__main__.py',
+    'ouroboros/unittest/main.py',
+    'ouroboros/unittest/result.py',
+    'ouroboros/unittest/runner.py',
+    'ouroboros/unittest/signals.py',
+    'ouroboros/unittest/suite.py',
+    'ouroboros/unittest/test/__main__.py',
+    'ouroboros/unittest/test/_test_warnings.py',
+    'ouroboros/unittest/test/dummy.py',
+    'ouroboros/unittest/test/support.py',
+    'ouroboros/unittest/util.py',
+    'ouroboros/urllib/error.py',
+    'ouroboros/urllib/response.py',
+    'ouroboros/venv/__main__.py',
+    'ouroboros/warnings.py',
+    'ouroboros/wave.py',
+    'ouroboros/wsgiref/headers.py',
+    'ouroboros/wsgiref/simple_server.py',
+    'ouroboros/wsgiref/util.py',
+    'ouroboros/wsgiref/validate.py',
+    'ouroboros/xml/dom/NodeFilter.py',
+    'ouroboros/xml/dom/domreg.py',
+    'ouroboros/xml/dom/expatbuilder.py',
+    'ouroboros/xml/dom/pulldom.py',
+    'ouroboros/xml/etree/ElementInclude.py',
+    'ouroboros/xml/etree/cElementTree.py',
+    'ouroboros/xml/parsers/expat.py',
+    'ouroboros/xml/sax/_exceptions.py',
+    'ouroboros/xml/sax/handler.py',
+    'ouroboros/xml/sax/saxutils.py',
+    'ouroboros/xml/sax/xmlreader.py',
+]
+
+
+def ouroboros_folder():
+    """Return the folder where the ouroboros repo was cloned."""
+    os.chdir(HERE)
+    os.chdir('../')
+    return os.path.join(os.getcwd(), 'ouroboros', 'ouroboros')
```

Comment:
Can't this be: `os.path.realpath(os.path.join(HERE, '../', 'ouroboros'))` ? (to avoid the `chdir`)